### PR TITLE
added readme

### PR DESCRIPTION
**Pull request by itsjzt**
😄 🉑

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ state.set({
 })
 
 state.subscribe({
-  counter: [document.querySelector('span'), 'innerText']
+  counter: [document.querySelector('span'), 'innerHTML']
```
Comment (sanatankc) [inline example.js]: innerHTML would change all node inside `span`, not that there is any for now, but still innerText is better.

**Comment by sanatankc** [comment]
Thanks for this pull merge @itsjzt. Just one small review before I merge this. 😄

File: example.js (hunk 1 above)
Comment (sanatankc) [reply example.js]: and also just live it as `'span'`. We don't need `document.querySelector('')` around it, we are selecting node inside the library. I made these changes earlier in the morning.

Comment (itsjzt) [reply example.js]: Ohhh! they were the merging issues! 😲 😞

**Comment by sanatankc** [comment]
I an merging this, will fix the issues later. 😄

**Pull request merged**